### DISCUSSION TITLE
Student ID declined because it's not english

### DISCUSSION THREAD

**Discussion by radstevee** [community/community#76894] [GitHub Education] [answered]
Hello, I just applied and got this response:
> The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

I submitted a picture of my german student ID. What do I do? I can't just get a translation of the ID.

**Comment by hackelite01** [comment]
hey @radstevee , No problem; mail me the ID card at hackelie.sup@gmail.com. I'll help to approve it with all procedures.

**Comment by sharabshrestha99** [comment]
I got the same response but as far as I know, I got rejected for the date. It's in our Nepal calendar how do we make it in AD?

**Comment by Usmani2085** [comment, marked as answer]
Hi, 
   You might need to provide additional documentation that displays your school name alongside your student ID, such as a transcript or an official letter from your school in English. This will supplement the German student ID[,](https://alightmotionzone.com/qr-codes/) ensuring your complete school name is visible in the submitted documents

Thanks

**Reply by ianchb** [reply]
Maybe so, but as a student in a Chinese school, I tried 120 times, uploading various materials, and was rejected.
The most interesting thing is that I passed the verification twice, but now I can't apply for the benefits again...
